case/src/main/kotlin/com/ritense/case/domain/CaseTab.kt

```diff
@@ -42,14 +42,16 @@ data class CaseTab(
     val createdOn: LocalDateTime? = LocalDateTime.now(),
 
     val createdBy: String? = null,
+    val showTasks: Boolean = false
 ) {
     constructor(
         id: CaseTabId,
         name: String?,
         tabOrder: Int,
         type: CaseTabType,
         contentKey: String,
-    ) : this(id, name, tabOrder, type, contentKey, LocalDateTime.now(), null)
+        showTasks: Boolean
+    ) : this(id, name, tabOrder, type, contentKey, LocalDateTime.now(), null, showTasks)
```

Comment:
```suggestion
    ) : this(id, name, tabOrder, type, contentKey, LocalDateTime.now(), null, false)
```
To avoid a breaking change